test/e2e/specs/all-features-confirmation/navbar/no-logo.spec.js

```diff
@@ -216,13 +205,36 @@ describe('Navbar ', function() {
         });
 
         it('should have a disabled link named "Disabled Link".', function () {
-            var dropdownOptions = element.all(by.css('.chaise-login-menu > li'));
+            var dropdownOptions = element.all(by.css('.username-display > div.dropdown-menu > *'));
```

Comment:
This should be a similar selector that we are using for the other dropdown menus. Above I suggested to do:
`recordsetsMenu.all(by.tagName('a'))`

This selector should work the same way if you get the loginDropdown first:
`element(by.css('.username-display > .dropdown-menu')).all(by.tagName(' a`))`